ci/avalon-direct-model-tests.yaml

```diff
@@ -15,15 +15,12 @@
 version: '3.5'
 
 services:
-  avalon-tests:
+  avalon-direct-model-tests:
     image: avalon-shell:${ISOLATION_ID}
-    build:
-      context: .
-    environment:
-      - http_proxy
-      - https_proxy
-      - no_proxy
     working_dir: "/project/avalon/"
+    volumes:
+      - ~/mywork/vars/keyfiles:/keyfiles
+      - ~/mywork/vars/profiles/mychannel_network_for_pysdk.json:/project/avalon/sdk/avalon_sdk/fabric/network.json
```

Comment:
Why to copy mywork directory contents to this yaml file which is for direct model tests?